corehq/apps/userreports/es/adapter.py

```diff
@@ -0,0 +1,44 @@
+from elasticsearch import NotFoundError
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.adapter import IndicatorAdapter
+from corehq.elastic import get_es_new
+
+
+class IndicatorESAdapter(IndicatorAdapter):
+
+    def __init__(self, config):
+        super(IndicatorESAdapter, self).__init__(config)
+        self.es = get_es_new()
+        self.table_name = get_table_name(config.domain, config.table_id)
+
+    def rebuild_table(self):
+        self.drop_table()
+        self.es.indices.create(index=self.table_name)
```

Comment:
I haven't done that yet. I imagine that it is going to be a little more complicated than that as it will probably be very dependent on expected # of docs per index and the size of those docs